### PR TITLE
implement function count_occurrences

### PR DESCRIPTION
**Pull request by Beliar24**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -14,4 +14,9 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    count = 0
+    for i in phrase:
+        if i.lower() == letter.lower():
+            count += 1
+
+    return count
```
Comment (Alkaponees) [inline app/main.py:22]: Try to write it in one line
use count()

**Review by Alkaponees — CHANGES_REQUESTED** [review]
Check my recommendation

File: app/main.py (hunk 1 above)
Comment (Beliar24) [reply app/main.py]: fixed

**Comment by Beliar24** [comment]
Changed

**Review by Alkaponees — APPROVED** [review]
Good Job!

**Pull request closed without merging**